hack/development-environment.md

```diff
@@ -0,0 +1,11 @@
+#### Development Environment
+If you want to develop the ingress controller locally, take the following steps.
+
+1. This guide assumes you are running in GKE.
+2. From this directory, apply the dbless config below to get kong running in your k8s cluster:
+
+    `kubectl apply -f dev-config.yaml`
+
+3. Run Kong Ingress Controller locally:
+
+    `bash start.sh kong`
```

Comment:
For step 2 and 3, assume that we are in the root directory.
Let's give the path as `hack/start.sh` and `hack/dev-config.yaml`.